src/Paket.Core/NuGetV2.fs

```diff
@@ -377,10 +377,15 @@ let rec private cleanup (dir : DirectoryInfo) =
     for file in dir.GetFiles() do
 
         let newName = Uri.UnescapeDataString(file.Name)
+        if newName.Contains "..\\" then
+          failwithf "Relative paths are not supported. Please tell the package author to fix the package to not use relative paths. The invalid file was '%s'" file.FullName
+        if newName.Contains "\\" then
+          traceWarnfn "File '%s' contains backslashes, probably because it wasn't properly packaged (for example with windows paths in nuspec on a unix like system). Please tell the package author to fix it." file.FullName
```

Comment:
If people start to complain about the new warnings you can remove this line (but I don't think they will as they would have noticed by now that paket3 wasn't working).